content/blog/tidymodels-2025-q1/index.Rmd

```diff
@@ -0,0 +1,235 @@
+  arrange(Latitude) |> 
+  ggplot(aes(x = Latitude)) + 
+  geom_point(data = ames, aes(y = Sale_Price), alpha = 1 / 5) +
+  geom_line(aes(y = .pred_quantile, col = format(.quantile_levels)), 
+            show.legend = FALSE, linewidth = 1.5) +
+  theme_bw()
+```
+
+For now, the new mode does not have many engines. We need to implement some performance statistics in the yardstick package before integrating these models into the whole tidymodels ecosystem. 
+
+In other news, we’ve added some additional neural network models based on some improvements in the brulee package. Namely, two-layer networks can be tuned for feed-forward networks on tabular data (using torch). 
+
+One other improvement has been simmering for a long time: the ability to exploit sparse data structures better. We’ve improved our `fit()` interfaces for the few model engines that can use sparsely encoded data. There is much more to come on this in a few months, especially around recipes, so stay tuned. 
+
+Finally, we’ve created a set of [checklists](https://parsnip.tidymodels.org/articles/checklists.html) that can be used when creating new models or engines. These are very helpful, even for us, since there is a lot of minutiae to remember. 
+
+## The tune Package
+
+This was a small maintenance release mostly related to parallel processing. Up to now, facilitated parallelism using the [foreach](https://cran.r-project.org/package=foreach) package. That package is mature but not actively developed, so we have been slowly moving toward using the [future](https://www.futureverse.org/packages-overview.html) package(s). 
+
+The [first step in this journey](https://www.tidyverse.org/blog/2024/04/tune-1-2-0/#modernized-support-for-parallel-processing) was to keep using foreach internally (but lean toward future) but to encourage users to move from directly invoking the foreach package and, instead, load and use the future package. 
+
+We’re now moving folks into the second stage. tune will now raise a warning when:
+
+- A parallel backend has been registered with foreach, and
+- No [`plan()`](https://future.futureverse.org/reference/plan.html) has been specified with future.
+
+This will allow users to transition their existing code to only future and allow us to update existing documentation and training materials. 
+
+We anticipate that the third stage, **removing foreach entirely**, will occur sometime before the 2025 Posit conference in September. 
+
+## The Next Cascade
+
+We are working hard on a few major initiatives that we plan on showing off at Posit conf. 
+
+First is integrated support for sparse **data**. The emphasis is on "data" because users can use sparse data frames _or_ the usual sparse matrix format. This is a big deal because it does not force you to convert non-numeric data into a numeric matrix format. Again, we’ll discuss this more in the future, but you should be able to use sparse data frames in parsnip, recipes, tune, etc. 
```

Comment:
sparse data frames are new thing, so maybe we should reword this block to avoid using that phrase. Maybe focusing more on the memory improvements

Reply:
How about "a data frame of sparse vectors"?